data/json/items/gun/22.json

```diff
@@ -23,6 +23,7 @@
     "valid_mod_locations": [ [ "accessories", 3 ], [ "barrel", 1 ], [ "bore", 1 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "muzzle", 1 ], [ "rail", 1 ], [ "sights", 1 ], [ "stock", 1 ], [ "underbarrel", 1 ] ],
     "magazines": [
       [ "22", [ "a180mag" ] ],
+      [ "46", [ "hk46bigmag" ] ],
```

Comment:
Magazine is very much the wrong shape. It's not necessary that every possible conversion is possible and perhaps even desirable that some are not.